src/ashley/api/serializers.py

```diff
@@ -0,0 +1,27 @@
+"""Serializer for ashley api."""
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+from ashley.context_mixins import get_current_lti_session
+
+User = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """Serializer for User model."""
+
+    role = serializers.SerializerMethodField()
+
+    class Meta:
+        model = User
+        fields = ["id", "public_username", "role"]
+        read_only_fields = ["id", "public_username", "role"]
+
+    def get_role(self, obj):
+        """Describes the role of the user"""
+        if obj.is_active:
+
+            lti_context = get_current_lti_session(self.context.get("request"))
+            return lti_context.get_user_roles(obj)
```

Comment:
I also suggest to use the plural form here:

```suggestion
    roles = serializers.SerializerMethodField()

    class Meta:
        model = User
        fields = ["id", "public_username", "roles"]
        read_only_fields = ["id", "public_username", "roles"]

    def get_roles(self, obj):
        """Returns the role list of the user within the current LTIContext."""
        if obj.is_active:

            lti_context = get_current_lti_session(self.context.get("request"))
            return lti_context.get_user_roles(obj)
```